### ISSUE TITLE
Lost issue notifications, when opening while no internet 

### ISSUE THREAD

**Issue by webmaster33** [gitskarios/Gitskarios#624]
I lost issue notifications, when opened Gitskarios, while there was no internet connection.

**Comment by alorma** [comment]
What you mean by lost?

**Comment by webmaster33** [comment]
1) start Gitskarios, while there is internet, and get notifications about issue updates. Exit. 

2) start Gitskarios by tapping the notification bar, while there is no internet connection. Exit. 

3) start Gitskarios again. The red notifications from Gitskarios are gone, and can't see what's new.

**Comment by alorma** [comment]
You can access notifications from side menu ;)

**Comment by webmaster33** [comment]
Yes, but they are not showing as new, after the no internet issue.

**Comment by rancidfrog** [comment]
I get notifications coming up randomly.
On the top right hand corner all is good,
![](https://s21.postimg.org/5i4v2j9t3/Screenshot_20161113_153728.png)
But, new notifications come up randomly,
![](https://s13.postimg.org/wh6ygeg3r/Screenshot_20161113_234637.png)
sometimes 5 days later even after I already checked them through right corner
